dev/devicelab/manifest.yaml

```diff
@@ -130,6 +130,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  embedded_android_views_integration_test:
+    description: >
+      Tests embedded Android views.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
```

Comment:
Based on our history of adding new tests, add:
```
flaky: true
```
for the first run. After a few green runs, remove the flaky marker.

Reply:
done